src/vstart.sh

```diff
@@ -564,13 +564,15 @@ EOF
 	    fi
 
 	    uuid=`uuidgen`
-	    echo "add osd$osd $uuid"
-	    $SUDO $CEPH_ADM osd create $uuid
+            echo "create osd with uuid ${uuid}"
```

Comment:
oh yeah, forgot about that ;) this was a quick and dirty patch on my tree that I thought I should share -- should have paid more attention to it. Will change and repush.